### PR TITLE
expose command to allow miners to set their storage price

### PR DESCRIPTION
**Pull request by whyrusleeping**

### CODE REVIEW

File: cmd/lotus-storage-miner/market.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	"fmt"
+
+	"github.com/filecoin-project/lotus/chain/types"
+	lcli "github.com/filecoin-project/lotus/cli"
+	"gopkg.in/urfave/cli.v2"
+)
+
+var setPriceCmd = &cli.Command{
+	Name:  "set-price",
+	Usage: "Set price that miner will accept storage deals at",
```
Comment (magik6k) [inline cmd/lotus-storage-miner/market.go]: Should say what the unit is to preempt the 20 questions we'll get about this

Comment (magik6k) [reply cmd/lotus-storage-miner/market.go]: (Optimally just accept FIL/GiB/Month as a logical unit)

**Comment by whyrusleeping** [comment]
@hannahhoward What are the units for price actually?

**Comment by hannahhoward** [comment]
@whyrusleeping 

Wow terrible docs on our part but:
Price per GiB per Epoch

**Comment by hannahhoward** [comment]
Meanwhile deals are proposed in FIL per byte per Epoch

**Comment by hannahhoward** [comment]
current calculations is

minPriceForPiece = ask.Price / pieceSizeInBytes * (1<<30)
storagePricePerEpoch must be less than or equal to minPriceForPiece in order for deal to proceed

**Comment by whyrusleeping** [comment]
This got merged as part of a different branch, but i'll update the help text in a separate PR

**Pull request closed without merging**